src/core/function/typed.js

```diff
@@ -147,6 +148,7 @@ export const createTyped = /* #__PURE__ */ factory('typed', dependencies, functi
     { name: 'RangeNode', test: isRangeNode },
     { name: 'SymbolNode', test: isSymbolNode },
 
+    { name: 'MapLike', test: isMapLike },
```

Comment:
At somepoint, the `Object | MapLike` is going to become just `MapLike`. I wonder at that point whether this type should then be renamed to `Scope`.

Reply:
This is exciting to me. Of course, the `math` namespace is could just be a parent `scope` where functions are defined.